src/metiq.py

```diff
@@ -1179,6 +1179,7 @@ def main(argv):
                     cache_audio=cache_audio,
```

Comment:
suggest renaming the patch to:

metiq: upgrade affine transformations to homographies

I understood immediately the "perspective transformation" term, but my reading suggests "homography" is a more common term in English